### PR TITLE
Latest files from Crowdin

### PR DESCRIPTION
**Pull request by zapbot**
Autogenerated PR from zapbot

### CODE REVIEW

File: src/org/zaproxy/zap/extension/coreLang/resources/Messages_pt_BR.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #X-Generator: crowdin.com
 # This file defines the default (English) variants of all of the internationalised messages
 
-coreLang.desc=Translations of the core language files
+coreLang.desc=Https\://crowdin. com/translate/owasp-zap/all/engb-ptbr\#2479213
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/coreLang/resources/Messages_pt_BR.properties]: This translation is not right :/

**Comment by kingthorin** [comment]
Good catch, shall we close it and revert via crowdin?

**Comment by thc202** [comment]
Was going to ask that, I think we should close.
(I reverted the change in the meantime.)

**Comment by kingthorin** [comment]
Thanks

**Comment by thc202** [comment]
Actually it was not reverted, it didn't save the changes. (I had to manually add the original string.)

Are you able to delete the suggestion of the REMOVED_USER? I'm not... :/

**Comment by kingthorin** [comment]
Done

**Comment by thc202** [comment]
Thanks!

**Pull request closed without merging**